### ISSUE TITLE
tl install complains of missing .tl.yml

### ISSUE THREAD

**Issue by JohnAlbin** [larowlan/tl#15]
Following the installation instructions in README.md:

```
$ tl install
Not configured Application is not confiured: please run the configure command.
File '/Users/jwilkins/.tl.yml' not found
```

"configured" is misspelled, fyi.

And then:

```
$ tl configure
PHP Notice:  Undefined index: url in phar:///usr/local/bin/tl/src/Connector/RedmineConnector.php on line 32
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/tl:0
PHP   2. require() /usr/local/bin/tl:10
PHP   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/tl/bin/tl.php:19
PHP   4. Larowlan\Tl\Application->doRun() phar:///usr/local/bin/tl/vendor/symfony/console/Application.php:126
PHP   5. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/src/Application.php:78
PHP   6. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
PHP   7. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:943
PHP   8. array_map() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
PHP   9. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
PHP  10. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1017
PHP  11. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
PHP  12. ReflectionClass->newInstanceArgs() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966
PHP  13. Larowlan\Tl\Connector\RedmineConnector->__construct() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966

Notice: Undefined index: url in phar:///usr/local/bin/tl/src/Connector/RedmineConnector.php on line 32

Call Stack:
    0.0025     224528   1. {main}() /usr/local/bin/tl:0
    0.0224     980416   2. require('phar:///usr/local/bin/tl/bin/tl.php') /usr/local/bin/tl:10
    0.0590    3750008   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/tl/bin/tl.php:19
    0.0639    4093256   4. Larowlan\Tl\Application->doRun() phar:///usr/local/bin/tl/vendor/symfony/console/Application.php:126
    0.0644    4122144   5. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/src/Application.php:78
    0.0644    4122488   6. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
    0.0645    4122928   7. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:943
    0.0645    4123416   8. array_map() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
    0.0645    4124072   9. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
    0.0645    4124216  10. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1017
    0.0645    4124496  11. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
    0.0725    4839760  12. ReflectionClass->newInstanceArgs() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966
    0.0725    4840000  13. Larowlan\Tl\Connector\RedmineConnector->__construct() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966

PHP Notice:  Undefined index: api_key in phar:///usr/local/bin/tl/src/Connector/RedmineConnector.php on line 33
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/tl:0
PHP   2. require() /usr/local/bin/tl:10
PHP   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/tl/bin/tl.php:19
PHP   4. Larowlan\Tl\Application->doRun() phar:///usr/local/bin/tl/vendor/symfony/console/Application.php:126
PHP   5. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/src/Application.php:78
PHP   6. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
PHP   7. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:943
PHP   8. array_map() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
PHP   9. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
PHP  10. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1017
PHP  11. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
PHP  12. ReflectionClass->newInstanceArgs() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966
PHP  13. Larowlan\Tl\Connector\RedmineConnector->__construct() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966

Notice: Undefined index: api_key in phar:///usr/local/bin/tl/src/Connector/RedmineConnector.php on line 33

Call Stack:
    0.0025     224528   1. {main}() /usr/local/bin/tl:0
    0.0224     980416   2. require('phar:///usr/local/bin/tl/bin/tl.php') /usr/local/bin/tl:10
    0.0590    3750008   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/tl/bin/tl.php:19
    0.0639    4093256   4. Larowlan\Tl\Application->doRun() phar:///usr/local/bin/tl/vendor/symfony/console/Application.php:126
    0.0644    4122144   5. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/src/Application.php:78
    0.0644    4122488   6. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
    0.0645    4122928   7. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:943
    0.0645    4123416   8. array_map() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
    0.0645    4124072   9. Symfony\Component\DependencyInjection\ContainerBuilder->resolveServices() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1015
    0.0645    4124216  10. Symfony\Component\DependencyInjection\ContainerBuilder->get() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:1017
    0.0645    4124496  11. Symfony\Component\DependencyInjection\ContainerBuilder->createService() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:490
    0.0725    4839760  12. ReflectionClass->newInstanceArgs() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966
    0.0725    4840000  13. Larowlan\Tl\Connector\RedmineConnector->__construct() phar:///usr/local/bin/tl/vendor/symfony/dependency-injection/ContainerBuilder.php:966
```

uh… The configure command seems to be complaining that the configuration is missing.

```
Creating new file
Enter your redmine URL: []
Enter your redmine URL: []https://redmine.previousnext.com.au     
Enter your redmine API Key: []    
```

I've heard of this. Can't figure out how to find it. ah. Just found it on `/my/account`. Should probably say that.

Now:

```
$ tl install
Executed 5 queries
```

**Comment by larowlan** [comment]
Thanks, up and running now?

**Comment by JohnAlbin** [comment]
Yep!

**Comment by larowlan** [comment]
I'll write some tests, the directory it uses is a container parameter so
can test

**Comment by larowlan** [comment]
Fixed in 0.0.11 please run tl self-update
